opentelemetry-zipkin/src/lib.rs

```diff
@@ -92,29 +92,35 @@ mod uploader;
 use async_trait::async_trait;
 use model::endpoint::Endpoint;
 use opentelemetry::{api::trace::TracerProvider, exporter::trace, global, sdk};
-use reqwest::Url;
 use std::error::Error;
 use std::net::SocketAddr;
 use std::sync::Arc;
+use http_client::http_types::{url, StatusCode};
+use http_client::{Response, Request, http_types};
+use std::ops::Deref;
 
 /// Default Zipkin collector endpoint
 const DEFAULT_COLLECTOR_ENDPOINT: &str = "http://127.0.0.1:9411/api/v2/spans";
 
 /// Default service name if no service is configured.
 const DEFAULT_SERVICE_NAME: &str = "OpenTelemetry";
 
+lazy_static::lazy_static! {
+    static ref DEFAULT_NOOP_HTTP_CLIENT: NoopHttpClient = NoopHttpClient::default();
```

Comment:
Having the default behavior be a blocking reqwest client may be preferable here. I think having a no-op would lead to a lot of users being confused why their spans are not showing up in zipkin (although it does mean a default reqwest dependency)

Reply:
Sorry forgot this is now an async api, if you're going with http_client this should probably be hyper as they impl it already.

Reply:
They do have a http client called H1 comes with the `http_client`. I also tried how easy it would be to implement `http_client::Client` for reqwest http clients. It actually takes quite some time even only accpet post requests.